src/topology/builder.rs

```diff
@@ -56,7 +53,7 @@ pub async fn build_pieces(
         .iter()
         .filter(|(name, _)| diff.sources.contains_new(&name))
     {
-        let (tx, rx) = mpsc::channel(1000);
+        let (tx, rx) = tokio::sync::mpsc::channel(1000);
```

Comment:
Do you mean to keep this on a tokio channel?

Reply:
It is intentional. The idea is to restrict the change to what was originally changed in #5868 so to bench against that and avoid improvement/regression of other things to mess with the results.

Reply:
Ah gotcha :+1: